### ISSUE TITLE
 Sync loss(Evaluation ) is 1.2 after 100 epoch

### ISSUE THREAD

**Issue by subhasis1990** [Rudrabha/Wav2Lip#279]
I am trying to train Wav2lip+GAN from scratch(LRS2 train) with the instruction you have provided. Even after 100 epoch Sync loss(Evaluation ) was 1.2.  So, I have manually changed Sync loss weight to 0.03 and train about 50 epochs. Now, my L1 loss is 0.023 and the Sync loss is 0.25. But the results are not good.

https://user-images.githubusercontent.com/16721547/120599481-33849500-c465-11eb-9560-afda317de684.mp4
 
Do I need to do any additional steps to make it work?

**Comment by Rudrabha** [comment]
So your sync-net itself has not converged? Sync-net often takes a very long time to converge in our experience. As long as it has not converged, it can't guide Wav2Lip to generate better results.

**Comment by subhasis1990** [comment]
I am using the pre-trained Syncnet() from this repo. Also, I was checking generated samples during training if the target pose is different from the identity image then generated samples are very bad. Do I need need to use a larger discriminator wt for training? Am I missing anything?

![0_3](https://user-images.githubusercontent.com/16721547/121538475-707ef780-ca22-11eb-94aa-dd682da0fdfc.jpg)
![1_2](https://user-images.githubusercontent.com/16721547/121538551-7ffe4080-ca22-11eb-97be-fc521fc13d42.jpg)

**Comment by GzuPark** [comment]
> I am using the pre-trained Syncnet(lipsync_expert.pth ) from this repo. Also, I was checking generated samples during training if the target pose is different from the identity image then generated samples are very bad. that can be the reason why the model is not converging. Do I need to use a larger discriminator wt for training? Am I missing anything?
> 
> ![0_3](https://user-images.githubusercontent.com/16721547/121538475-707ef780-ca22-11eb-94aa-dd682da0fdfc.jpg)
> ![1_2](https://user-images.githubusercontent.com/16721547/121538551-7ffe4080-ca22-11eb-97be-fc521fc13d42.jpg)

It looks totally overfitting. Half below of the 3rd row is the same as the 1st row.

**Comment by lifemo** [comment]
I meet the situation too. 
Firstly, average_sync_loss  drops to about 1 and is stable. But after 180 epochs, it grew to dozens. 
Have you solved this problem ?

**Comment by Linyxxxxx** [comment]
> I meet the situation too. Firstly, average_sync_loss drops to about 1 and is stable. But after 180 epochs, it grew to dozens. Have you solved this problem ?

Same problem to you. Have you solved it?

**Comment by amostayed** [comment]
> So your sync-net itself has not converged? Sync-net often takes a very long time to converge in our experience. As long as it has not converged, it can't guide Wav2Lip to generate better results.

So, what is a good convergence point for ColorSyncNet pretraining? I have trained the model (for LRS2) for about 100 epochs with the 0.0001 learning rate and did not manage to get the Sync loss (both train & eval) below 0.42. I also tried LR scheduling but could not improve. So, from your experience what would be a sufficient number of epochs to train to achieve convergence and what is the loss values should I expect at that point? Any suggestions will be greatly appreciated.

**Comment by HaSaKiYasuooo** [comment]
> > 那么你的同步网络本身还没有收敛？Sync-net 通常需要很长时间才能收敛到我们的经验中。只要它没有收敛，就无法引导Wav2Lip产生更好的结果。
> 
> 那么，ColorSyncNet 预训练的一个好的收敛点是什么呢？我已经以 0.0001 的学习率训练了模型（针对 LRS2）约 100 个时期，但未能使同步损失（训练和评估）低于 0.42。我也尝试过LR调度但无法改进。那么，根据您的经验，需要训练多少纪元才能实现收敛，此时我应该期望损失值是多少？任何建议将不胜感激。
![image](https://github.com/Rudrabha/Wav2Lip/assets/87591236/dbe4eceb-64e5-485e-bf35-f5d87f0c9ca7)
is　this 　a normal phenomenon　？

**Comment by thd-ux** [comment]
When comparing with other benchmark models, do we need to ensure that all models are trained on the same dataset?
Thanks!
